client/src/components/Details.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReadMore from './ReadMore'
+
+class Details extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showReadMore: false
+    }
+  }
+
+  handleReadMoreClick(event) {
+    event.preventDefault();
+    this.setState({
+      showReadMore: true
+    });
+  }
```

Comment:
I see you have a clickHandler method but I don't see where it's being used or being passed as props. Is this being used yet?

Reply:
Not being used yet, but in this neighborhood service component, there is a link titled 'read more about this neighborhood' that conditionally renders more information on click. I still need to determine which component or subcomponent this link will be located. I am trying to form somewhat of a template that will be refactored in a future branch.

Reply:
This is okay, because the function is complete, and it isn't making a breaking changes.